package.json

```diff
@@ -60,6 +60,7 @@
     "solhint": "^3.3.7",
     "solhint-plugin-prettier": "^0.0.5",
     "solidity-coverage": "^0.7.21",
+    "solmate": "^6.5.0",
```

Comment:
where are do we use this?

Reply:
In the mock erc20 contract.

Reply:
Removed the dependency in [0479366](https://github.com/quest-chains/contracts/pull/13/commits/047936694a1112467b7bc10f266c891dedc2f79f)